modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-preparing-playbook-machine-{context}"]
+= Preparing the machine to run the playbook
+
+Before you can add compute machines that use Red Hat Enterprise Linux as the
+operating system to a {product-title} {product-version} cluster, you must
+prepare a machine to run the playbook from. This machine is not part of the
+cluster but must be able to access it.
+
+.Procedure
+
+. Ensure that the `kubeconfig` file for the cluster and the installation program
+that you used to install the cluster are on the machine. One way to accomplish
+this is to use the same machine that you used to install the cluster.
+
+. Configure the machine to access all of the RHEL hosts that you plan to use as
+compute machines. You can use any method that your company allows, including a
+bastion with an SSH proxy or a VPN.
+
+. Configure a user on the machine that you run the playbook on that has SSH
+access to all of the RHEL hosts.
++
+[IMPORTANT]
+====
+If you use SSH key-based authentication, you must manage the key with an
+SSH agent.
+====
+
+. Install the OpenShift Command-line Interface (CLI), commonly known as `oc`, on
+the machine.
+
+. Install Ansible 2.7 and the `OpenShift-Ansible` package on the machine:
```

Comment:
In the example build this is item 5 and it seems to be more related to registering to RHSM than installing packages.

Reply:
@mtnbikenc, do you not need to register the hosts before you obtain the packages?

Reply:
Yes, they need to be registered before installing packages.  Here is an example of what I was trying to say before (some details omitted):
```
    5. Register the control host with RHSM and enable the required repositories:
        a. If you have not, register the machine with RHSM:
        b. Pull the latest subscription data from RHSM:
        c. List the available subscriptions:
        d. In the output for the previous command, find the pool ID for an OpenShift Container Platform subscription and attach it:
        e. Enable the repositories required by OpenShift Container Platform Branch Build:
    6. Install OpenShift-Ansible and other required packages on the machine:
        a. Install OpenShift-Ansible
              # yum install openshift-ansible openshift-clients jq
```

Reply:
@mtnbikenc, if they've already registered the machine and attached the pool, would 5a-5d be required?